docs/UnitySimulationHelpInformation.md

```diff
@@ -1,24 +1,24 @@
-# Unity Simulation Help and Information 
-Guide for navigating issues in Unity Simulation that can block work flow in using the SynthDet Sample Project.
+# Unity Simulation help and information 
+This is a guide for navigating issues in Unity Simulation that can block your workflow when using the SynthDet Sample Project.
 
-## Run in USim Window
+## Run in Unity Simulation window
 
-This section will explain the Run in USim window and the meanings behind the parameters that are needed to create a Unity Simulation run
+This section explains the Run in Unity Simulation window and the definitions of the parameters that are needed to create a Unity Simulation run:
 
 <img src="images/USimRunWindow.PNG" align="middle"/>
 
 ### Parameters
-* Run name is used by Unity Simulation to label the run being executed on a node  
-* Path to build zip is used for location of the linux player build of the target project, this will be unzipped and ran on a Unity Simulation node
-* Scale factor range is a graph mapping the scale factor values from 0-1 range
-* Scale factor steps is the number of samples between 0-1 that we will simulate on
-* Another way to think of scale factor range and steps is a change in the graph that min and max values in the graph range
+* Run Name is used by Unity Simulation to label the run being executed on a node  
+* Path to build zip is used for the location of the Linux player build of the target project: the user unzips this and runs it on a Unity Simulation node
+* Scale Factor Range is a graph mapping the scale factor values in the 0-1 range
+* Scale Factor Steps is the number of samples between 0-1 that the user simulates on
+* Another way to think of Scale Factor Range and Scale Factor Steps is a change in the graph that min and max values in the graph range
```

Comment:
I'm not sure what this bullet point means. Can you clarify?

Reply:
@Saurav-D How about this:
>Scale Factor Steps is the number of scale factors that will be executed in this run. SynthDet will generate hundreds of images for each step, so this number also controls the amount of images generated in the dataset.

Reply:
I've added this text in the latest commit.